src/components/Layout/CheckboxCustom.tsx

```diff
@@ -29,13 +27,9 @@ const CheckboxCustom = ({ formValue, label, required = false, colorScheme = 'bra
     <FormControl isInvalid={!!errors[formValue]}>
       <Flex alignItems='center'>
         <Checkbox {...register(formValue, validationRules)} colorScheme={colorScheme} me={2.5} />
-        <FormLabel display='flex' mb={0} fontWeight='normal' color={textColor} fontSize='sm'>
+        <FormLabel display='flex' mb={0} fontWeight='normal' fontSize='sm'>
           <Box>{label}</Box>
-          {required && (
-            <Text color={textColor} ml={1}>
-              *
-            </Text>
-          )}
+          {required && <Text ml={1}>*</Text>}
```

Comment:
This is not needed. FormControl already adds this if you set the `isRequired` prop.